docs/general/serializers.md

```diff
@@ -369,3 +369,19 @@ class PostSerializer < ActiveModel::Serializer
   end
 end
 ```
+
+## Overriding default serializer
```

Comment:
I'd rather change it to `## Overriding association serializer lookup` as this should only works for the relationships of this specific serializer.